src/client/styles/custom.less

```diff
@@ -88,6 +88,7 @@
 @modal-background-color: rgba(242, 242, 242, 0.97);
 
 // z-indexs
+@ant-message-z-index: 1400;
```

Comment:
What happens if we have both message and image gallery open at the same time?

Reply:
good question, I didn't know if image gallery should be behind or infront of the message, so i just set it the same